sass/components/_categorical-display.scss

```diff
@@ -348,6 +348,19 @@
     }
 }
 
+.smartcharts-desktop {
+    &.smartcharts-480 {
```

Comment:
you could fuse the bottom code block with `&.smartcharts-480, &.smartcharts-900`

I'm still passing to QA to see if it solves that they asked for

Reply:
@brucebinary but their width is different, its `450px` for `.smartcharts-480` and `550px` for `.smartcharts-900`